### DISCUSSION TITLE
company-files: tilde slash ("~/") suggests contents of %AppData% directory on Windows

### DISCUSSION THREAD

**Discussion by cwfoo** [company-mode/company-mode#1281] [Q&A] [answered]
When I type `~/` in Emacs on Windows, company-files will show the contents of the `%AppData%` directory (`C:\Users\myusername\AppData\Roaming` in Windows Vista and later) as completion candidates. Is this the intended behavior? `~/` in Windows does not have the same meaning as `~/` in Unix-like systems.

**Comment by dgutov** [comment, marked as answer]
Apparently so, yes. `$HOME` is AppData in Windows.
